base/system/winlogon/shutdown.c

```diff
@@ -4,6 +4,7 @@
  * FILE:            base/system/winlogon/shutdown.c
  * PURPOSE:         System shutdown dialog
  * PROGRAMMERS:     alpha5056 <alpha5056@users.noreply.github.com>
```

Comment:
```suggestion
 * PROGRAMMERS:     Edward Bronsten <tanki.alpha5056@gmail.com>
```
Extracted author data from PR #458, you probably can fix this in formatting commit.